editor/debugger/editor_profiler.cpp

```diff
@@ -351,6 +352,9 @@ void EditorProfiler::_update_frame() {
 		for (int j = m.categories[i].items.size() - 1; j >= 0; j--) {
 			const Metric::Category::Item &it = m.categories[i].items[j];
 
+			if ((it.internal == it.total) && !display_internal_profiles->is_pressed() && m.categories[i].name == "Script Functions") {
```

Comment:
```suggestion
			if (it.internal == it.total && !display_internal_profiles->is_pressed() && m.categories[i].name == "Script Functions") {
```
Or put parentheses around all, uniformity